chia/daemon/client.py

```diff
@@ -46,7 +61,9 @@ async def _get(self, request: WsRpcMessage) -> WsRpcMessage:
         request_id = request["request_id"]
         self._request_dict[request_id] = asyncio.Event()
         string = dict_to_json_str(request)
-        asyncio.create_task(self.websocket.send(string))
+        if self.websocket is None:
```

Comment:
Won't the `asyncio.create_task(self.websocket.send(string))` throw anyway in this case?

Reply:
why would it throw?

Reply:
if `self.websocket is none` - then I would expect `asyncio.create_task(self.websocket.send(string))` to throw so one approach is to not bother with the "if" check and just let it throw. Which I assume the previous code does.

Reply:
I'll just do that.

Reply:
correction, one of the linters wants it.